src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -165,6 +165,16 @@ static void reset_byte_stream(void* arg, grpc_error* error);
 // GRPC_EXPERIMENTAL_DISABLE_FLOW_CONTROL
 bool g_flow_control_enabled = true;
 
+namespace grpc_core {
+TestOnlyGlobalHttp2TransportInitCallback test_only_init_callback = nullptr;
```

Comment:
Is it possible to add this in a way that isn't test-only? It seems there's no structure in gRPC that would already know the set of all active transports, so I'm not sure how else we could get a count. It's worth considering if we can eliminate test-only code in non-test paths.

Reply:
Suggestions welcome :)

Reply:
As discussed, I've spent zero time in the transport code, so I defer to you. My only alternative thought was that we could count resource users indirectly, which should decrease by 2 when the endpoints shut down, but that's a hack.